### PR TITLE
mysql_protocol v2.0

### PR DESCRIPTION
**Pull request by slegrand45**

### CODE REVIEW

**Comment by dsheets** [comment]
Seems Travis missed this PR...

**Comment by yallop** [comment]
I'll close and reopen to trigger a build.

**Comment by yallop** [comment]
Thanks!

File: packages/mysql_protocol/mysql_protocol.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+license: "LGPL"
+dev-repo: "https://github.com/slegrand45/mysql_protocol.git"
+build: [
+  [make "all"]
+]
+install: [make "install"]
+remove: [make "deinstall"]
+depends: [
+  "ocamlbuild" {build}
+  "ocamlfind"
```
Comment (dsheets) [inline packages/mysql_protocol/mysql_protocol.2.0/opam]: This should probably be a `{build}` dependency in future releases.

**Pull request merged**